Mathlib/Data/List/DropRight.lean

```diff
@@ -247,4 +247,22 @@ theorem rtakeWhile_idempotent : rtakeWhile p (rtakeWhile p l) = rtakeWhile p l :
   rtakeWhile_eq_self_iff.mpr fun _ => mem_rtakeWhile_imp
 #align list.rtake_while_idempotent List.rtakeWhile_idempotent
 
+
+lemma rdrop_add.{u} {α : Type u} {l : List α} (i j : ℕ) :
```

Comment:
```suggestion
lemma rdrop_add {l : List α} (i j : ℕ) :
```
The type `α` already is available as a variable. In fact `l` is also available, but as explicit argument. So adding it manually to make it implicit is fine.